### ISSUE TITLE
In-App messages

### ISSUE THREAD

**Issue by vadimwe** [laravel-notification-channels/onesignal#106]
Do you plan to implement In-App Messages feature?

**Comment by LKaemmerling** [comment]
I don't think we can implement this, because it is not available in the API reference: https://documentation.onesignal.com/reference  

Maybe if there are endpoints available we can look into this.

**Comment by vadimwe** [comment]
Seems now there is support
